Source/WebCore/dom/Element.h

```diff
@@ -35,6 +35,9 @@
 #include "ScrollTypes.h"
 #include "ShadowRootMode.h"
 #include "SimulatedClickOptions.h"
+#include "TrustedHTML.h"
+#include "TrustedScript.h"
+#include "TrustedScriptURL.h"
```

Comment:
A way to get rid of these imports is to make a separate setAttribute and setAttributeNS function which doesn't take the variant (basically the original function) and only export those. The implementation can just call your variant version. That way the webkit legacy doesn't need the concrete types.